### PR TITLE
Feat 16 navigation small screen

### PR DESCRIPTION
**Pull request by SinchanaMS**
Created the Submenu and Hamburger Navigation components.

### Preview:

1. Submenu:

image

2. Hamburger:

https://user-images.githubusercontent.com/94738532/161286104-6a18dad0-f755-407c-ac0b-66477f76b482.mp4

### CODE REVIEW

File: stylesheets/components/navigation/navigation.css [hunk 1]
```diff
@@ -1,13 +1,120 @@
...
+    }
+
+    .ham-list{
+        display: none;
+    }
+    
+    .ham-list.mobile-view{
+        display: flex;
+        flex-direction: column;
+        gap: 12px;
```
Comment (Ananyamadhu08) [inline stylesheets/components/navigation/navigation.css]: Can use rem instead of px

**Review by Ananyamadhu08 — APPROVED** [review]
Your UI is amazing it looks very professional! Your code is readable and understandable, good variable naming.  You can go ahead and merge it.

File: stylesheets/components/navigation/navigation.css (hunk 1 above)
Comment (manojsarna) [inline stylesheets/components/navigation/navigation.css]: Can use rems if the values are greater than 10 px

**Review by manojsarna — APPROVED** [review]
Overall code looks great !!

Do Check the comments

**Review by yashodaAgarwal — APPROVED** [review]
Your UI looks good. 
Code is readable and clean

**Comment by SinchanaMS** [comment]
Thank you for the reviews. Merging to temp-dev.

**Pull request merged**